source/includes/stackpath/_workloads.md

```diff
@@ -21,39 +21,60 @@ curl -X GET \
 {
   "data": [
     {
-      "id": "8531b1ba-1c31-41ab-b0c4-8f65798e5ed7",
       "name": "container-workload",
-      "stackId": "4f246958-595d-4ead-ae56-a88cff334b97",
+      "stackId": "a397735b-66d2-49ba-b665-e77acb944481",
       "slug": "container-workload",
       "version": "1",
-      "created": "2020-11-26T16:07:42.788665965Z",
+      "created": "2021-02-22T17:32:27.202428099Z",
       "type": "CONTAINER",
-      "commands": "/bin/sh -c \"sleep 50\"",
       "network": "default",
-      "specs": "SP-1",
       "cpu": "1",
       "memory": "2Gi",
       "isRemoteManagementEnabled": false,
-      "image": "redis",
+      "image": "nginx",
       "addImagePullCredentialsOption": false,
-      "containerUsername": null,
-      "containerServer": null,
-      "containerEmail": null,
-      "environmentVariableKey": "key1",
-      "environmentVariableValue": "new-val",
-      "deploymentName": "my-deployment",
-      "deploymentPops": [
-        "MIA",
-        "ATL",
-        "DFW"
+      "environmentVariables": [
+        {
+          "key": "USER",
+          "value": "my-user"
+        }
       ],
-      "enableAutoScaling": true,
-      "cpuUtilization": 56,
-      "minInstancesPerPop": 1,
-      "maxInstancesPerPop": 2,
+      "secretEnvironmentVariables": [
+        {
+          "key": "PASSWORD",
+          "value": "[REDACTED]"
```

Comment:
We should explain the redacted and the fact they need to repost this info to not remove the field.. Same in the get section

Reply:
Also updated all the attribute sections [here](https://github.com/cloudops/cloudmc-api-docs/pull/307/commits/449603ceaff570ede613097481fdb46a6dbac7a2#diff-009b71b1c0eb5e3fe379e21d571a83b999e8661251e350ecb0efb9319d7a992eR135)